### PR TITLE
CLICKHOUSE-2720: progress on website and reference

### PR DESCRIPTION
**Pull request by blinkov**

### CODE REVIEW

**Comment by robot-metrika-test** [comment]
Can one of the admins verify this patch?

File: doc/reference/ru/operations/tips.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Размер блока в 1024 Кб подходит для всех конфигураций RAID.
+Никогда не указывайте слишком маленький или слишком большой размер блока.
+
+
+На SSD можно использовать RAID-0.
+Вне зависимости от использования RAID, всегда используйте репликацию для безопасности данных.
+
+Включите NCQ с длинной очередью. Для HDD стоит выбрать планировщик CFQ, а для SSD — noop. Не стоит уменьшать настройку readahead.
+На HDD стоит включать write ahread.
```
Comment (alexey-milovidov) [inline doc/reference/ru/operations/tips.rst]: ?

File: doc/reference/ru/operations/tips.rst [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+
+Файловая система
+----------------
+
+Ext4 — самый проверенный вариант, стоит указывать опции монтирования ``noatime,nobarrier``.
+XFS также подходит, но не так тщательно протестирована в сочитании с ClickHouse.
```
Comment (alexey-milovidov) [inline doc/reference/ru/operations/tips.rst]: Typo.

File: doc/reference/ru/operations/tips.rst [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+Вероятно вы уже используете ZooKeeper для других целей. Можно использовать ту же инсталляцию ZooKeeper, если она не сильно перегружена.
+
+Лучше использовать свежую версию ZooKeeper, как минимум 3.5. Версия в стабильных дистрибутивах Linux может быть устаревшей.
+
+С настройками по умолчанию, ZooKeeper является бомбой замедленного действия:
+
+    Сервер ZooKeeper не будет удалять файлы со старыми снепшоты и логами при использовании конфигурации по умолчанию (см. autopurge), это является ответственностью оператора.
+
+Эту бомбу нужно обезвредить.
```
Comment (alexey-milovidov) [inline doc/reference/ru/operations/tips.rst]: You could link https://github.com/yandex/ClickHouse/blob/master/doc/administration/zookeeper.md for further information about ZooKeeper configuration.

File: doc/reference/ru/operations/tips.rst (hunk 1 above)
Comment (blinkov) [inline doc/reference/ru/operations/tips.rst]: Fixed.

File: doc/reference/ru/operations/tips.rst (hunk 2 above)
Comment (blinkov) [inline doc/reference/ru/operations/tips.rst]: Fixed.

File: doc/reference/ru/operations/tips.rst (hunk 3 above)
Comment (blinkov) [inline doc/reference/ru/operations/tips.rst]: Obviously that was the plan, but we first need to migrate it to sphinx too.

**Comment by blinkov** [comment]
It'd be nice to merge it asap, because without 3569fbb the "Edit this page" links in docs lead to 404.

**Pull request merged**